wormhole/wormhole_test.go

```diff
@@ -747,11 +749,38 @@ func (ts *testRelayServer) handleConn(c net.Conn) {
 func TestClient_relayURL_default(t *testing.T) {
 	var c Client
 
-	DefaultTransitRelayURL = "tcp:transit.magic-wormhole.io:8001"
-	p := c.relayURL().Proto
-	if p != "tcp" {
-		t.Error(fmt.Sprintf("invalid protocol, expected tcp, got %v", p))
-	}
+	expectedProto := "tcp"
+	expectedHost := "transit.magic-wormhole.io"
+	expectedPort := "8001"
```

Comment:
while not functionally relevant, I think this should be 4001:
```suggestion
	expectedPort := "4001"
```